set.mm

```diff
@@ -44315,9 +44314,7 @@ proper class (see for example ~ iprc ).  (Contributed by NM,
     $( An equality theorem for the maps-to notation.  (Contributed by Thierry
        Arnoux, 30-May-2020.) $)
     mpteq12df $p |- ( ph -> ( x e. A |-> B ) = ( x e. C |-> D ) ) $=
```

Comment:
I am inclined to keep the name mpteq12df, but, of course with the proof of mpteq12d.  The f suffix indicates the use of a not-free hypothesis, and is in this case closer to mpteq12f than to mpteq12.
This means the proof and credits are taken from mpteq12d, the name changes to mpteq12df, and mpteq12d is deleted.

Reply:
Actually I noticed the two proofs are identical too, should I switch credits anyway?

Reply:
Because the proofs are identical, the older credits prevail.

Reply:
Done